### PR TITLE
Create sprint

### PR DESCRIPTION
**Pull request by bic-marz**
The flex layout, in the issue card, sometimes works sometimes doesn't, when switching from/to dark-mode

### CODE REVIEW

File: project-canvas/src/components/DetailView/Components/StoryPointsEstimateMenu.tsx [hunk 1]
```diff
@@ -60,63 +60,75 @@ export function StoryPointsEstimateMenu({
   }
 
   return (
```
Comment (oussema18) [inline project-canvas/src/components/DetailView/Components/StoryPointsEstimateMenu.tsx]: the Loader wont disappear and keep loading when setting the storyPoints 0

**Review by oussema18 — CHANGES_REQUESTED** [review]

File: project-canvas/src/components/DetailView/Components/StoryPointsEstimateMenu.tsx (hunk 1 above)
Comment (bic-marz) [reply project-canvas/src/components/DetailView/Components/StoryPointsEstimateMenu.tsx]: fixed

File: project-canvas/src/components/CreateIssue/Fields/PrioritySelect.tsx [hunk 2]
```diff
@@ -27,24 +27,37 @@ export function PrioritySelect({
...
-              label: priority.name,
-            }))
-          : []
-      }
-      searchable
-      clearable
-      {...form.getInputProps("priority.id")}
-    />
+    <Tooltip
+      label="Priority cannot be selected for this issue type"
```
Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/PrioritySelect.tsx:31]: when the issue type is not selected the message Priority cannot be selected for this issue type  is not clear . it should be choose an issue type first

Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/PrioritySelect.tsx:31]: label={
        form.getInputProps("type").value
          ? "Priority cannot be selected for this issue type"
          : "Please select an issue type first"
      }

File: project-canvas/src/components/CreateIssue/Fields/ReporterSelect.tsx [hunk 3]
```diff
@@ -5,20 +5,18 @@ import { SelectItem } from "../SelectItem"
...
-  currentUser?: User
   assignableUsers?: User[]
   isLoading: boolean
 }) {
   return (
     <Select
       label="Reporter"
-      placeholder={currentUser?.displayName || "Select a Reporter"}
-      nothingFound="No Options"
+      placeholder="Choose reporter"
```
Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/ReporterSelect.tsx:18]: Please select an issue type first"

File: project-canvas/src/components/CreateIssue/Fields/SprintSelect.tsx [hunk 4]
```diff
@@ -39,22 +39,35 @@ export function SprintSelect({
...
-              label: sprint.name,
-            }))
-          : []
-      }
-      searchable
-      clearable
-      {...form.getInputProps("sprintId")}
-    />
+    <Tooltip
+      label="Sprint cannot be selected for this issue type"
```
Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/SprintSelect.tsx:43]: Please select an issue type first"

File: project-canvas/src/components/CreateIssue/Fields/EpicSelect.tsx [hunk 5]
```diff
@@ -24,31 +24,49 @@ export function EpicSelect({
...
-              label: epic.summary,
-            }))
-          : []
-      }
-      searchable
-      clearable
-      {...form.getInputProps("epic")}
-    />
+    <Tooltip
+      label="Epic cannot be selected for this issue type"
```
Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/EpicSelect.tsx:38]: Please select an issue type first"

File: project-canvas/src/components/CreateIssue/Fields/StoryPointsEstimateInput.tsx [hunk 6]
```diff
@@ -21,13 +21,25 @@ export function StoryPointsEstimateInput({
...
-    <NumberInput
-      min={0}
-      label="Story Point Estimate"
-      defaultValue={null as unknown as number}
-      precision={3}
-      disabled={isDisabled}
-      {...form.getInputProps("storyPointsEstimate")}
-    />
+    <Tooltip
+      label="Story Point Estimate cannot be defined for this issue type"
```
Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/StoryPointsEstimateInput.tsx:25]: Please select an issue type first"

File: project-canvas/src/components/CreateIssue/Fields/DueDatePicker.tsx [hunk 7]
```diff
@@ -9,29 +10,42 @@ export function DueDatePicker({
...
-      onChange={(value) => {
-        form.getInputProps("dueDate").onChange(value)
-        if (
-          value &&
-          form.getInputProps("startDate").value &&
-          form.getInputProps("startDate").value > value
-        )
-          form.setFieldValue("startDate", null as unknown as Date)
+    <Tooltip
+      label="Due Date cannot be picked for this issue type"
```
Comment (oussema18) [inline project-canvas/src/components/CreateIssue/Fields/DueDatePicker.tsx:21]: Please select an issue type first"

**Review by oussema18 — CHANGES_REQUESTED** [review]

**Review by oussema18 — APPROVED** [review]

File: project-canvas/src/components/CreateIssue/Fields/PrioritySelect.tsx (hunk 2 above)
Comment (bic-marz) [reply project-canvas/src/components/CreateIssue/Fields/PrioritySelect.tsx:31]: Good point

**Pull request merged**